<?xml version="1.0" encoding="UTF-8"?><testrun name="A380Tests" project="AircraftSimulator" tests="36" started="36" failures="0" errors="1" ignored="0">
  <testsuite name="asgn2Tests.A380Tests" time="0.059">
    <testcase name="testA380DepartureTimeLessThan0" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testBookingsEmptiesBusiness" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsAvailableBusiness" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testGetPassengersWhen1" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testBookingsEmptiesPremium" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testFlyPassengers" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsNotAvailablePremium" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testCancelBooking" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testHasPassenger" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testGetPassengersWhenMultiple" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testGetBookings" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testEconomyUpgradeBookings" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testPremiumUpgradeBookings" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testA380DepartureLessThan0" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testBusinessUpgradeBookings" classname="asgn2Tests.A380Tests" time="0.002"/>
    <testcase name="testConfirmBooking" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsAvailableFirst" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testAircraft" classname="asgn2Tests.A380Tests" time="0.022">
      <error>java.lang.Exception: Unexpected exception, expected&lt;asgn2Aircraft.AircraftException&gt; but was&lt;java.lang.Error&gt;&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:28)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.lang.Error: Unresolved compilation problem: 
	Unhandled exception type AircraftException
&#13;
	at asgn2Tests.A380Tests.testAircraft(A380Tests.java:111)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)&#13;
	... 15 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="testA380FlightCodeNull" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testSeatsNotAvailableFirst" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsAvailableEconomy" classname="asgn2Tests.A380Tests" time="0.005"/>
    <testcase name="testFlightEmpty" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsNotAvailableBusiness" classname="asgn2Tests.A380Tests" time="0.002"/>
    <testcase name="testFirstUpgradeBookings" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testInitialState" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testA380EconomyLessThan0" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testBookingsEmptiesEconomy" classname="asgn2Tests.A380Tests" time="0.001"/>
    <testcase name="testFlightFull" classname="asgn2Tests.A380Tests" time="0.006"/>
    <testcase name="testSeatsNotAvailableEconomy" classname="asgn2Tests.A380Tests" time="0.01"/>
    <testcase name="testA380BusinessLessThan0" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testHasPassengerWhenMultiplePassengers" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testFinalState" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testSeatsAvailablePremium" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testA380PremiumLessThan0" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="test2ConfirmBookingS" classname="asgn2Tests.A380Tests" time="0.0"/>
    <testcase name="testA380FirstLessThan0" classname="asgn2Tests.A380Tests" time="0.002"/>
  </testsuite>
</testrun>
